spark/src/test/java/org/opensearch/sql/spark/dispatcher/SparkQueryDispatcherTest.java

```diff
@@ -780,7 +787,7 @@ void testDispatchDescribeIndexQuery() {
     StartJobRequest expected =
         new StartJobRequest(
             query,
-            "TEST_CLUSTER:index-query",
+            "TEST_CLUSTER:non-index-query",
```

Comment:
@dai-chen @vamsi-amazon we could process DESC SKIPPING INDEX in interactive/batch query, right?

Reply:
yes, all SHOW/DESC index can actually be treated as direct query